packages/firestore/test/integration/api/server_timestamp.test.ts

```diff
@@ -209,6 +209,14 @@ apiDescribe('Server Timestamps', (persistence: boolean) => {
   });
 
   it('can return previous value', () => {
+    // The following update includes an update of the nested map "deep", which
```

Comment:
"The following update includes an update" sounds wrong. Maybe you meant "The following test includes an update"?

Also "As such" doesn't work here. I think what you're trying to say is that the implementation of the update call is to split it into two mutations ...

Reply:
Removed "as such" and replaced with "This update".